File: Kattis/python/different.py
def different_problem(num1, num2) -> int:
    return abs(num1 - num2)

def main():
    while True:
        try:
            num1, num2 = map(int, input().split())
        except EOFError:
            break
        print(different_problem(num1, num2))
    
if __name__ == "__main__":
    main()

File: Kattis/go/different.go
package main

import (
	"fmt"
	"math"
)

func main() {
	for {
		var num1, num2 float64
		_, err := fmt.Scanln(&num1, &num2)
		if err != nil {
			return
		}
		fmt.Printf("%d\n", int(math.Abs(num1-num2)))
	}
}
